statistics_api/definitions.py

```diff
@@ -73,7 +70,9 @@ def get_ip_address():
 if ip:
     allowed_hosts.append(ip)
 
+allowed_hosts.append("0.0.0.0")
```

Comment:
Hvordan påvirker dette koden når den går live og hva er behovet for å tillate alle ipv4 adresser?